spyder/plugins/updatemanager/workers.py

```diff
@@ -9,10 +9,12 @@
 import logging
 import os
 import os.path as osp
+import platform
 import shutil
 import sys
 from time import sleep
 import traceback
+from typing import TypedDict
 from zipfile import ZipFile
 
 # Third party imports
```

Comment:
```suggestion
# Third party imports
import packaging.version
```

This is necessary to improve annotations below.

Reply:
How about this?
```suggestion
# Third party imports
from packaging.version import parse, Version
```